static/js/types/app/nl_interface_types.ts

```diff
@@ -83,3 +85,10 @@ export interface UserMessageInfo {
   msgList: string[];
   showForm: boolean;
 }
+
+export interface EntityPvConfig {
+  entities: NamedNode[];
+  properties: NamedNode[];
+  propertyValues: Record<string, Record<string, Node[]>>;
```

Comment:
Nit: Can you add a quick comment explaining this typing? Is this entity dcid -> property dcid -> value node?

Reply:
good call, done